pixl_ehr/pyproject.toml

```diff
@@ -17,6 +17,7 @@ dependencies = [
     "psycopg2==2.9.5",
     "azure-identity==1.12.0",
     "azure-storage-blob==12.14.1",
+    "pyarrow==14.0.1",
```

Comment:
is pyarrow needed here? I thought it would have been in pixl core which we're installing

Reply:
Apparently yes! See #213 
```
E           ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
E           A suitable version of pyarrow or fastparquet is required for parquet support.
E           Trying to import the above resulted in these errors:
E            - Missing optional dependency 'pyarrow'. pyarrow is required for parquet support. Use pip or conda to install pyarrow.
E            - Missing optional dependency 'fastparquet'. fastparquet is required for parquet support. Use pip or conda to install fastparquet.
```